sdl_android/src/main/java/com/smartdevicelink/proxy/interfaces/IProxyListenerBase.java

```diff
@@ -356,4 +364,16 @@
 	public void onSendHapticDataResponse(SendHapticDataResponse response);
 
 	public void onOnRCStatus(OnRCStatus notification);
+
+	public void onPublishAppServiceResponse(PublishAppServiceResponse response);
+
+	public void onGetAppServiceDataResponse(GetAppServiceDataResponse response);
+
+	public void onGetFileResponse(GetFileResponse response);
+
+	public void onPerformAppServiceInteractionResponse(PerformAppServiceInteractionResponse response);
+
+	public void onOnAppServiceData(OnAppServiceData notification);

```

Comment:
Probably remove the first "on" from `onOnAppServiceData` and `onOnSystemCapabilityUpdated`